### ISSUE TITLE
Community-mantained fork

### ISSUE THREAD

**Issue by sebasgarcep** [redux-offline/redux-offline#117]
This project became extremely popular because it solves a very important problem: making apps offline-first. Seeing as how @jevakallio currently doesn't have the time to take care of this project we should consolidate all our efforts into a single community-mantained fork until the current mantainer is available again or he gives away ownership of the project.

I've seen @sorodrigo keeps a fork as well as @McPo, but I'm not sure if they are willing to carry the burden of mantaining this project from now forward.

tl;dr We should fork this project and move over to the fork.

**Comment by sorodrigo** [comment]
I agree and all the help is welcomed and appreciated. I'd be happy to help maintain a community driven fork. Leaving always the door open for when @jevakallio is available again.
I'd also like to include @j8seangel into this thread given that the fork under my account has been maintained by both of us.

**Comment by wacii** [comment]
I offered to help maintain this library. When I didn't hear back I started writing my own version, but this is a much better idea.

I'd be willing to help.

**Comment by j8seangel** [comment]
Sounds good to join efforts and make this library more useful. I'm in!

**Comment by bsouthga** [comment]
Would moving/forking it under https://github.com/reactjs make sense?

**Comment by wacii** [comment]
I think creating our own organization would be less hassle. Especially if we eventually wanted to peel off bits into separate modules. Like effects reconcilers or other config-related things.

**Comment by sorodrigo** [comment]
@bsouthga @wacii an organization seems good to me. How about `@redux-offline/redux-offline` also having a gitter (or similar) channel would be nice.

**Comment by sebasgarcep** [comment]
@sorodrigo Seems like a good idea!

**Comment by sorodrigo** [comment]
I have created the organization https://github.com/redux-offline 😄

**Comment by sebasgarcep** [comment]
@sorodrigo We need to enable issues in the new repo.

**Comment by sorodrigo** [comment]
@sebasgarcep Oops! It's enabled now 😉

**Comment by jevakallio** [comment]
Hi all. 

Apologies for total radio silence on this project. Can't really give a good explanation, except that burnout is a bitch and it makes it incredibly hard to do even the simplest things, such as responding to GitHub issues.

I'm happy to support migrating this project to more active maintainers. It appears the best way forward is to migrate the repo to the redux-offline org to keep online links etc working, and give you admin bits to it.

@sorodrigo are you still interested in taking the ownership to the org you created?

**Comment by sebasgarcep** [comment]
@jevakallio You don't have to excuse yourself. Open source is a favor not a responsability. We are all grateful for what you started.

**Comment by sorodrigo** [comment]
@jevakallio hey!! No worries man! I love what you did here! About taking the ownership, that would be awesome. 😄

**Comment by jevakallio** [comment]
@sorodrigo 

I couldn't migrate this to the redux-offline organization due to
1. Not having access to create repositories in that org
2. A repository by the same name already existing in that org

I also couldn't give you administrative access to jevakallio/redux-offline, due it it being privately owned. 

To get around this, I've now migrated this repo to a temporary org https://github.com/redux-offline-team, where I have invited you as a member, and given you administrative rights to this repo.

You should be now able to either rename the redux-offline-team org to redux-offline, or move your fork and transfer this to the redux-offline org.

Thanks for maintaining an active interest in this repo! I trust you'll do a great job with it. There are others, such as @fabriziomoscon who might be interested in helping out.

Good luck!

**Comment by ghost** [comment]
@jevakallio you monster! Thanks for your work.

**Comment by sorodrigo** [comment]
@jevakallio Thanks a lot again, have been on vacations. Will get things moving soon. :)

**Comment by ghost** [comment]
Can we switch also the stars? It's a pity to loose all that stars.

**Comment by piranna** [comment]
> Can we switch also the stars? It's a pity to loose all that stars.

If we sync (merge back) the `redux-offline` organization repo with this one, we could remove that one and we'll only loose its stars and issues, not the ones of this repo. I think it's the best thing, move back things from the `redux-offline` to the `redux-offline-team` one and after that, remove `redux-offline` team and rename this one to it.

**Comment by seanpowell** [comment]
Hey gang - Where did we leave this?  Is it officially moved to: https://github.com/redux-offline/redux-offline?  Or is it staying here and changing ownership?

Definitely interested in keeping this moving and can help move/setup/other legwork if needed.  Thanks for keeping this moving everyone.

**Comment by sorodrigo** [comment]
Hey! The merge back is imminent, we’re wrapping up a release on the other repo to make the merge easier.

**Comment by piranna** [comment]
What's the final ststus?

**Comment by wacii** [comment]
The community fork has been merged. We're discussing moving this repo over to the redux-offline organization [here](https://github.com/redux-offline/redux-offline/issues/66), but that shouldn't have any noticeable effect on this repo.

**Comment by ghost** [comment]
We have loose the stars?

**Comment by piranna** [comment]
> We have loose the stars?

i see 3925
